src/test/java/net/pms/configuration/FormatConfigurationRegressionTest0.java

```diff
@@ -10,966 +10,699 @@
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "MIMETYPE_AUTO"+ "'", str0.equals("MIMETYPE_AUTO"));
-
   }
 
   @Test
   public void test42() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test42"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.VP7;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "vp7"+ "'", str0.equals("vp7"));
-
   }
 
   @Test
   public void test43() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test43"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.THREEGPP;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "3gp"+ "'", str0.equals("3gp"));
-
   }
 
   @Test
   public void test44() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test44"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.MJPEG;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "mjpeg"+ "'", str0.equals("mjpeg"));
-
   }
 
   @Test
   public void test45() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test45"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.BMP;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "bmp"+ "'", str0.equals("bmp"));
-
   }
 
   @Test
   public void test46() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test46"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.TTA;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "tta"+ "'", str0.equals("tta"));
-
   }
 
   @Test
   public void test47() throws Throwable {
+    java.lang.String str0 = net.pms.configuration.FormatConfiguration.AAC_LC;
 
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test47"); }
+    // Regression assertion (captures the current behavior of the code)
+    org.junit.Assert.assertTrue("'" + str0 + "' != '" + "aac-lc"+ "'", str0.equals("aac-lc"));
+  }
 
-    java.lang.String str0 = net.pms.configuration.FormatConfiguration.AAC;
+  @Test
+  public void testAACMain() throws Throwable {
+    java.lang.String str0 = net.pms.configuration.FormatConfiguration.AAC_MAIN;
 
     // Regression assertion (captures the current behavior of the code)
-    org.junit.Assert.assertTrue("'" + str0 + "' != '" + "aac"+ "'", str0.equals("aac"));
-
+    org.junit.Assert.assertTrue("'" + str0 + "' != '" + "aac-main"+ "'", str0.equals("aac-main"));
```

Comment:
@Sami32 When I make something I don't make it thinking like that. I separate concepts that I think shouldn't be reliant on each other. MediaInfo is on possible parser, FFmpeg is another, for audio files so is JAudioTagger. I made a parser for Real Audio, and we might find another external or make another internal parser in the future. 

In my experience software that doesn't assume a lot about what other parts is doing work best. The parser should do it's part: parse at much and as accurate as it can. Other code should do its part, in this situation handle what the parsers might output.

Don't forget that there's many renderer configurations that doesn't even use MediaInfo. I think that blindly focusing on what MediaInfo outputs is a mistake.

But, as I said above, I don't remember my reason for including ```aac-main``` now, as it's been too long since I did it. It might have been that I saw that it could be a parsing result (maybe RealAudio can have it?), it could be that I thought it should be there for "logical reasons" or it could be that I followed some line of thought that turns out to be irrelevant. The problem is that I don't remember.

I didn't mean to say that it has to stay, I'm actually unsure as I can't remember the reason for including it, I just wanted to say that what MediaInfo can parse or not isn't the deciding factor for me. It seems to be for you, since you use the fact that MediaInfo can't parse it as an argument for why it will never be used.